core/src/main/java/nu/yona/server/exceptions/ResourceBasedException.java

```diff
@@ -31,27 +33,17 @@ void onContextStarted(ContextRefreshedEvent event)
 	}
 
 	private static final long serialVersionUID = 8031973645020363969L;
-	/** Holds the parameters for the exception message. */
-	private final Object[] parameters;
-	/** Holds the message id. */
+	private final String[] messageArgs;
 	private final String messageId;
-	/** Holds the HTTP response code to be used. */
 	private final HttpStatus statusCode;
 	private static Translator translator;
 
-	/**
-	 * Constructor.
-	 * 
-	 * @param statusCode The status code of the exception.
-	 * @param messageId The ID of the exception in the resource bundle
-	 * @param parameters The parameters for the message
-	 */
-	protected ResourceBasedException(HttpStatus statusCode, String messageId, Object... parameters)
+	protected ResourceBasedException(HttpStatus statusCode, String messageId, Object... messageArgs)
 	{
 		super(messageId);
 
 		this.messageId = messageId;
-		this.parameters = parameters;
+		this.messageArgs = serializeMessageArgs(messageArgs);

```

Comment:
Actually not. Also in our context, we require late translation. We are logging all exceptions in English, even though we return them to the user in Dutch or even Chinese (later).

I'm creating PR with a fix.